### ISSUE TITLE
Client 3.2 VFS "Couldn't create placeholder info"

### ISSUE THREAD

**Issue by jmigual** [nextcloud/desktop#3198]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

### Expected behaviour
Nextcloud synchronizes properly when using the **VFS** feature

### Actual behaviour
Multiple files show the error "Couldn't create placeholder info" (see attached picture)

<p align="center">
<img width="584" alt="Screenshot 2021-04-23 212609" src="https://user-images.githubusercontent.com/6493651/115920510-948a7780-a47a-11eb-8418-545e5b12c247.png">
</p>

### Steps to reproduce
1. Sync a new folder in PC
2. After a while the issue occurs.

### Client configuration
Client version: 3.2.0 (Windows)

Operating system: Windows

OS language: Catalan and English (issue happens with both languages so it doesn't look to be language-dependent)

Installation path of client: `C:\Program Files\Nextcloud`

### Server configuration

Nextcloud version: `21.0.1`

Storage backend (external storage): No external storage

### Logs

Please use Gist (https://gist.github.com/) or a similar code paster for longer
logs.

1. Client logfile: [20210423_2050_owncloud.log.3.zip](https://github.com/nextcloud/desktop/files/6367689/20210423_2050_owncloud.log.3.zip)

I am wondering whether this could be related to a big number of files (>200000) synchronized at once.

**Comment by jmigual** [comment]
Okay I think I found some clues. In my case I have files with same name but different capitalization. So I have the file `001.jpg` and the file `001.JPG` so I guess that there is some confusion coming from there.

**Comment by allexzander** [comment]
> Okay I think I found some clues. In my case I have files with same name but different capitalization. So I have the file `001.jpg` and the file `001.JPG` so I guess that there is some confusion coming from there.

@jmigual 

When you're not using the VFS and syncing the same folder, will there be errors displayed too? I guess there should be errors about "local name clash" without the VFS.

**Comment by jmigual** [comment]
@allexzander Yes you are right. I've disabled VFS and created a small example (I ended up fixing the whole folder by renaming all the files) and indeed it gives me an error about a conflict with names. 

I've created two files `test.md` and `TEST.md` and it gives me the following error:

![image](https://user-images.githubusercontent.com/6493651/116692314-7235b400-a9bc-11eb-8f6c-3ab595fe09b6.png)

Which translates for "Cannot save the file test\test.md because there is a conflict with a local name".
